--- abinit-8.10.2/configure	2019-01-17 03:37:58.000000000 -0800
+++ patch-files/configure	2019-03-21 15:10:36.822219028 -0700
@@ -27677,9 +27677,7 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  abi_gpu_cuda_serial="yes"
-fi
+abi_gpu_cuda_serial="yes"
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: ${abi_gpu_cuda_serial}" >&5
@@ -27712,11 +27710,12 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  abi_gpu_cuda_old="no"
-else
+#if ac_fn_c_try_link "$LINENO"; then :
+#  abi_gpu_cuda_old="no"
+#else
   abi_gpu_cuda_old="yes"
-fi
+#fi
+abi_gpu_cuda_old="no"
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: ${abi_gpu_cuda_old}" >&5
