--- abinit-8.8.4/configure	2018-07-26 23:24:43.000000000 -0700
+++ patch-files/configure	2018-10-01 18:28:20.237327881 -0700
@@ -27654,9 +27654,10 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  abi_gpu_cuda_serial="yes"
-fi
+#if ac_fn_c_try_link "$LINENO"; then :
+#  abi_gpu_cuda_serial="yes"
+#fi
+abi_gpu_cuda_serial="yes"
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: ${abi_gpu_cuda_serial}" >&5
@@ -27689,11 +27690,12 @@
   return 0;
 }
 _ACEOF
-if ac_fn_c_try_link "$LINENO"; then :
-  abi_gpu_cuda_old="no"
-else
+#if ac_fn_c_try_link "$LINENO"; then :
+#  abi_gpu_cuda_old="no"
+#else
   abi_gpu_cuda_old="yes"
-fi
+#fi
+abi_gpu_cuda_old="no"
 rm -f core conftest.err conftest.$ac_objext \
     conftest$ac_exeext conftest.$ac_ext
   { $as_echo "$as_me:${as_lineno-$LINENO}: result: ${abi_gpu_cuda_old}" >&5
