--- magma-2.4.0/make.inc-examples/make.inc.mkl-gcc	2018-06-25 15:23:47.000000000 -0700
+++ patch-files/make.inc.mkl-gcc	2018-10-01 18:34:49.446225180 -0700
@@ -17,7 +17,7 @@
 # Note that NVIDIA no longer supports 1.x cards, as of CUDA 6.5.
 # See http://developer.nvidia.com/cuda-gpus
 #
-#GPU_TARGET ?= Kepler Maxwell Pascal
+GPU_TARGET = Kepler Maxwell Pascal
 
 # --------------------
 # programs
